### ISSUE TITLE
Crashing due to unicode or something

### ISSUE THREAD

**Issue by dukkk** [Codeusa/Borderless-Gaming#430]
mNão é possível converter o carácter Unicode \uDE18 no índice 77 para a página de códigos especificada.EncoderFallbackExceptionᕠ  em System.Text.EncoderExceptionFallbackBuffer.Fallback(Char charUnknown, Int32 index)
   em System.Text.EncoderFallbackBuffer.InternalFallback(Char ch, Char*& chars)
   em System.Text.UTF8Encoding.GetBytes(Char* chars, Int32 charCount, Byte* bytes, Int32 byteCount, EncoderNLS baseEncoder)
   em System.Text.EncoderNLS.GetBytes(Char* chars, Int32 charCount, Byte* bytes, Int32 byteCount, Boolean flush)
   em System.Text.EncoderNLS.GetBytes(Char[] chars, Int32 charIndex, Int32 charCount, Byte[] bytes, Int32 byteIndex, Boolean flush)
   em System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder)
   em System.IO.StreamWriter.Write(Char[] buffer, Int32 index, Int32 count)
   em System.IO.TextWriter.WriteLine(String value)
   em System.IO.TextWriter.SyncTextWriter.WriteLine(String value)
   em System.Console.WriteLine(String value)
   em BorderlessGaming.Logic.Windows.Native.GetWindowTitle(IntPtr hWnd) em E:\Dev\BG\Borderless-Gaming\BorderlessGaming.Logic\Windows\Native.cs:line 223
   em BorderlessGaming.Logic.Models.ProcessDetails..ctor(Process p, IntPtr hWnd) em E:\Dev\BG\Borderless-Gaming\BorderlessGaming.Logic\Models\ProcessDetails.cs:line 31
   em BorderlessGaming.Logic.Windows.Native.QueryProcessesWithWindows(Action`1 callback, List`1 windowPtrSet) em E:\Dev\BG\Borderless-Gaming\BorderlessGaming.Logic\Windows\Native.cs:line 436
   em BorderlessGaming.Logic.Core.ProcessWatcher.<UpdateProcesses>d__20.MoveNext() em E:\Dev\BG\Borderless-Gaming\BorderlessGaming.Logic\Core\ProcessWatcher.cs:line 206
--- Fim do rastreio da pilha da localização anterior em que a excepção foi emitida ---
   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em BorderlessGaming.Logic.Core.ProcessWatcher.<Watch>d__15.MoveNext() em E:\Dev\BG\Borderless-Gaming\BorderlessGaming.Logic\Core\ProcessWatcher.cs:line 61
--- Fim do rastreio da pilha da localização anterior em que a excepção foi emitida ---
   em System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_1(Object state)
   em System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)
   em System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   em System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   em System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   em System.Threading.ThreadPoolWorkQueue.Dispatch()
   em System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()

**Comment by kevinvanrijn** [comment]
Getting the same error here. Using whatever the current version on Steam is.

Here's the same error again in English:
[UnhandledException_30-01-19.crash](https://github.com/Codeusa/Borderless-Gaming/files/2900605/UnhandledException_30-01-19.txt)

I rolled back to the version before anything related to Rainway was introduced and haven't crashed yet since. ( https://steamdb.info/depot/388081/history/?changeid=3547394 )

That's quite a range of commits to go through though. So hopefully someone else can narrow down what change caused this.

```
download_depot 388080 388081 683910069201558777
download_depot 388080 229004 5220958916987797232
```

^ Exact files I rolled back to. First line is Borderless Gaming. (Shows up as 9.4.9) Second is the .NET Redist.
